src/Common/Elf.h

```diff
@@ -54,7 +54,8 @@ class Elf final
     const char * end() const { return mapped + elf_size; }
     size_t size() const { return elf_size; }
 
-    /// Obtain build id from PT_NOTES section of program headers. Return empty string if does not exist.
+    /// Obtain build id from SHT_NOTE of section headers (fallback to PT_NOTES section of program headers).
```

Comment:
Why we should extract build id from debug info file?

Reply:
The question is answered by
"But we have to compare Build ID to check that debug info corresponds to the same executable"
comment in collectSymbolsFromELF